apps/app/src/mocks/miragejs/orm/registry/Registry.ts

```
@@ -0,0 +1,35 @@
+    if (this.has(name)) {
+      throw new Error(`Item '${name}' is already registered`);
+    }
+    return super.set(name, item);
+  }
+}
+
+/**
+ * Registry class for storing user-defined model classes, factories, and identity managers
+ */
+export default class Registry {
+  public readonly factories: RegistryItemManager<FactoryInstance<any>>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)